### PR TITLE
fix: implements sticky footer

### PR DESCRIPTION
**Pull request by dzucconi**
I think there was a ticket associated with this but I can't find it now.

### CODE REVIEW

File: src/v2/Apps/Components/HorizontalPadding.tsx [hunk 1]
```diff
@@ -1,36 +1,14 @@
...
-    `};
-  ${p =>
-    // @ts-expect-error STRICT_NULL_CHECK
-    p.px[1] &&
-    css`
-      padding-right: ${// @ts-expect-error STRICT_NULL_CHECK
-      space(p.px[1])}px;
-      padding-left: ${// @ts-expect-error STRICT_NULL_CHECK
-      space(p.px[1])}px;
-    `};
```
Comment (dzucconi) [inline src/v2/Apps/Components/HorizontalPadding.tsx:29]: All this was doing was just manually allowing padding overwrites — which doesn't need to be done if you use `defaultProps`

File: src/v2/Apps/Components/AppContainer.tsx [hunk 2]
```diff
@@ -4,6 +4,9 @@ import React from "react"
 
 interface AppContainerProps extends BoxProps {}
 
+/**
+ * Handles the max-width of the application. That's it!
+ */
```
Comment (damassi) [inline src/v2/Apps/Components/AppContainer.tsx:9]: ty 👍

File: src/v2/Apps/Components/HorizontalPadding.tsx (hunk 1 above)
Comment (damassi) [reply src/v2/Apps/Components/HorizontalPadding.tsx:29]: heh, nice refactor

**Comment by damassi** [comment]
🎉

**Pull request merged**